docs/monitoring.md

```diff
@@ -435,6 +435,7 @@ set of sinks to which metrics are reported. The following instances are currentl
 * `executor`: A Spark executor.
 * `driver`: The Spark driver process (the process in which your SparkContext is created).
 * `shuffleService`: The Spark shuffle service.
+* `applicationMaster`: The Spark application master on YARN.
 
```

Comment:
I think it would be better to clarify as "The Spark ApplicationMaster when running on YARN."

Reply:
Thanks, updated accordingly.